libkbfs/kbpki_client.go

```diff
@@ -243,3 +243,21 @@ func (k *KBPKIClient) loadUnverifiedKeys(ctx context.Context, uid keybase1.UID)
 	[]keybase1.PublicKey, error) {
 	return k.config.KeybaseService().LoadUnverifiedKeys(ctx, uid)
 }
+
+// GetCurrentUserIfLoggedIn returns the current username and uid from
+// kbpki.GetCurrentUserInfo if the user is logged in or empty ones
+// if they are not logged in.
+func GetCurrentUserIfLoggedIn(ctx context.Context, kbpki KBPKI, isPublic bool) (libkb.NormalizedUsername, keybase1.UID, error) {
```

Comment:
1) Done
2) Done
3) Just using `KBPKI.GetCurrentUserInfo` works in this case, using that instead.

`GetCurrentUsernameIfLoggedIn` is used from libfuse and libdokan.